src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,62 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Task {
+    private static final DateTimeFormatter INPUT_FORMAT =
```

Comment:
Should there be spacing between your fields as logical units within a block should be separated by one blank line?

Reply:
Great observation, I did not think of that. I will incorporate the required adjustments in the next commit!